common/src/main/java/com/gregtechceu/gtceu/api/block/MaterialBlock.java

```diff
@@ -56,7 +56,7 @@ public MaterialBlock(Properties properties, TagPrefix tagPrefix, Material materi
     public static BlockColor tintedColor() {
         return (state, reader, pos, tintIndex) -> {
             if (state.getBlock() instanceof MaterialBlock block) {
-                if (tintIndex == 1 && block.material.getMaterialSecondaryRGB() != -1) {
+                if ((tintIndex == 1 || tintIndex == -111) && block.material.getMaterialSecondaryRGB() != -1) {
```

Comment:
the 2nd color layer for material blocks shouldn't have bloom.

Reply:
it doesn't make bloom effect